pcsx2/DEV9/AdapterUtils.cpp

```diff
@@ -509,3 +513,63 @@ std::vector<IP_Address> AdapterUtils::GetDNS(ifaddrs* adapter)
 	return collection;
 }
 #endif
+bool AdapterUtils::GetAdapterMAC(u8 mac[6], const char* adapter_name)
```

Comment:
```suggestion
bool AdapterUtils::GetAdapterMAC(std::array<u8, 6> mac, const char* adapter_name)
```

Using C-style arrays in arguments are possible, but the inflict an invalid sense of type safety:
https://godbolt.org/z/6GnWf4WxK